### PR TITLE
chore: `db-tools` Make target for MDBX debugging tools

### PR DESCRIPTION
**Pull request by shekhirin**
### Build
```console
➜  reth (alexey/db-tools) ls db-tools
"db-tools": No such file or directory (os error 2)

➜  reth (alexey/db-tools) make db-tools                                                             
Building MDBX debugging tools...
Run "db-tools/mdbx_stat" for the info about MDBX db file.
Run "db-tools/mdbx_chk" for the MDBX db file integrity check.
```

### Run tools
```console
➜  reth (alexey/db-tools) db-tools/mdbx_stat ~/Library/Application\ Support/reth/sepolia/db/mdbx.dat
mdbx_stat v0.12.6-0-gc019631a (2023-04-29T21:30:35+03:00, T-44de01dd81ac366a7d37111eaf72726edebe5528)
Running for /Users/shekhirin/Library/Application Support/reth/sepolia/db/mdbx.dat...
Status of Main DB
  Pagesize: 16384
  Tree depth: 0
  Branch pages: 0
  Leaf pages: 0
  Overflow pages: 0
  Entries: 0

➜  reth (alexey/db-tools) db-tools/mdbx_chk ~/Library/Application\ Support/reth/sepolia/db/mdbx.dat 
mdbx_chk v0.12.6-0-gc019631a (2023-04-29T21:30:35+03:00, T-44de01dd81ac366a7d37111eaf72726edebe5528)
Running for /Users/shekhirin/Library/Application Support/reth/sepolia/db/mdbx.dat in 'read-only' mode...
Traversal b-tree by txn#3...
Scanning @MAIN for sub-database(s)...
No error is detected, elapsed 0.018 seconds
```

### Build again, shouldn't crash
```console
➜  reth (alexey/db-tools) make db-tools                                                             
Building MDBX debugging tools...
Run "db-tools/mdbx_stat" for the info about MDBX db file.
Run "db-tools/mdbx_chk" for the MDBX db file integrity check.
```

### Check `git status`, everything should be clean
```console
➜  reth (alexey/db-tools) git status
On branch alexey/db-tools
Your branch is up to date with 'origin/alexey/db-tools'.

nothing to commit, working tree clean
```

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -147,4 +149,23 @@ endef
...
 	cargo clean
 	rm -rf $(BIN_DIR)
-	rm -rf $(EF_TESTS_DIR)
\ No newline at end of file
+	rm -rf $(EF_TESTS_DIR)
+
+# Compile MDBX debugging tools
+.PHONY: db-tools
+db-tools:
+	@echo "Building MDBX debugging tools..."
```
Comment (onbjerg) [inline Makefile:157]: use `$(info)`, e.g. `$(info Building MDBX debugging tools)`

https://www.gnu.org/software/make/manual/make.html#Make-Control-Functions

File: Makefile [hunk 2]
```diff
@@ -147,4 +149,23 @@ endef
...
+    # `IOARENA=1` silences benchmarking info message that is printed to stderr
+	@cd crates/storage/libmdbx-rs/mdbx-sys/libmdbx && make IOARENA=1 tools > /dev/null
+	@mkdir -p $(DB_TOOLS_DIR)
+	@cd crates/storage/libmdbx-rs/mdbx-sys/libmdbx && \
+		mv mdbx_chk $(FULL_DB_TOOLS_DIR) && \
+		mv mdbx_copy $(FULL_DB_TOOLS_DIR) && \
+		mv mdbx_dump $(FULL_DB_TOOLS_DIR) && \
+		mv mdbx_drop $(FULL_DB_TOOLS_DIR) && \
+		mv mdbx_load $(FULL_DB_TOOLS_DIR) && \
+		mv mdbx_stat $(FULL_DB_TOOLS_DIR)
```
Comment (onbjerg) [inline Makefile:167]: can use `$(MAKE) -C crates/storage/libmdbx-rs/mdbx-sys/libmdbx tools`, see https://github.com/paradigmxyz/reth/blob/main/crates/storage/libmdbx-rs/mdbx-sys/libmdbx/GNUmakefile

then afterwards move them

File: Makefile [hunk 3]
```diff
@@ -147,4 +149,23 @@ endef
...
+    # `IOARENA=1` silences benchmarking info message that is printed to stderr
+	@cd crates/storage/libmdbx-rs/mdbx-sys/libmdbx && make IOARENA=1 clean > /dev/null
```
Comment (onbjerg) [inline Makefile:169]: use `$(MAKE) -C ... clean` like above

File: Makefile [hunk 4]
```diff
@@ -147,4 +149,23 @@ endef
...
+	@echo "Run \"$(DB_TOOLS_DIR)/mdbx_stat -h\" for the info about MDBX db file."
+	@echo "Run \"$(DB_TOOLS_DIR)/mdbx_chk -h\" for the MDBX db file integrity check."
```
Comment (onbjerg) [inline Makefile]: `$(info ...)`

**Review by onbjerg — CHANGES_REQUESTED** [review]

File: Makefile (hunk 2 above)
Comment (shekhirin) [reply Makefile]: nice

**Review by onbjerg — APPROVED** [review]
lgtm

**Pull request merged**